editor/plugins/style_box_editor_plugin.cpp

```diff
@@ -32,6 +32,13 @@
 
 #include "editor/editor_scale.h"
 
+bool StyleBoxPreview::grid_preview_enabled = true;
+
+void StyleBoxPreview::_grid_preview_pressed() {
+	grid_preview_enabled = grid_preview->is_pressed();
```

Comment:
You wouldn't need to check `is_pressed()` if you connected the `toggled` signal (it passes the pressed state as argument and generally should be used for such cases).

Reply:
This is a static variable, to keep it the same for next times the inspector is instantiated

Reply:
No the static variable is fine. What I mean is that you can connect `toggled` signal instead of `pressed`, so your function would be
```C++
void StyleBoxPreview::_grid_preview_toggled(p_pressed) {
	grid_preview_enabled = p_pressed;
```
`toggled` signal is more suited here.

Reply:
Makes sense, I realize I have the habit of doing this because original the UI system in Godot did not have  toggled signal and you could check the state from the button directly, but eventually because deferred connections were often needed this no longer worked in some cases so I added the toggled signal, but I often forget its there.

Reply:
Okay, fixed now.